app/views/personal_access_tokens/_personal_access_tokens_tab.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="tab-pane" id="personal_access_tokens">
   <%= react_component('PersonalAccessTokens', {
     url: api_user_personal_access_tokens_path(user_id: @user),
-    canCreate: authorized_for(controller: 'api/v2/personal_access_tokens', auth_action: :create)
+    canCreate: authorized_for(controller: 'api/v2/personal_access_tokens', auth_action: :create, user_id: User.current.to_param)
```

Comment:
Wait, this is not correct, is it? we don't want the user_id to be current user here, but `@user` (the user we are editing)

Reply:
Thanks! PR updated.